resources/dicts/events/misc/general.json

```diff
@@ -1,3722 +1,3943 @@
-        "event_text": "m_c starts bringing a pretty acc_singular everywhere {PRONOUN/m_c/subject} {VERB/m_c/go/goes}, and r_c loudly complains that {PRONOUN/r_c/subject} {VERB/r_c/want/wants} one, too.",
-        "new_accessory": ["WILD", "PLANT"],
+        "event_text": "Whilst out collecting herbs, m_c stumbles upon a o_c_n warrior attempting to pack a wound with leaves. Their movements are slow and uncoordinated - without help, they'll soon be hunting with their ancestors. It would be so easy to just... walk past. But {PRONOUN/m_c/subject} can't. It's just a few pieces of cobweb from the branches and a few moments of {PRONOUN/m_c/poss} time... nobody has to know.",
         "m_c": {
-            "age": ["kitten"]
+            "status": ["medicine cat", "medicine cat apprentice"]
         },
-        "r_c": {
-          "age":["kitten"],
-          "trait": ["bullying", "impulsive", "attention-seeker"]
+        "other_clan": {
+            "changed": 1
+        }
+    },
+    {
+        "event_id": "gen_misc_war_medcat_woundedenemy_neg",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 20,
+        "event_text": "Whilst out collecting herbs, m_c stumbles upon a o_c_n warrior attempting to pack a wound with leaves. Their movements are slow and uncoordinated - without help, they'll soon be hunting with their ancestors. It would be so easy to just... walk past. But {PRONOUN/m_c/subject} can't. A few berries placed into their jaws with some soothing words... and that's one less o_c_n cat to deal with.",
+        "m_c": {
+            "status": ["medicine cat", "medicine cat apprentice"]
         },
-        "relationships": [
-          {
-            "cats_to": ["m_c"],
-            "cats_from": ["r_c"],
-            "mutual": true,
-            "values": ["dislike", "jealousy"],
-            "amount": 10
-          },
-          {
-           "cats_to": ["m_c"],
-            "cats_from": ["r_c"],
-            "mutual": true,
-            "values": [ "platonic", "comfort"],
-            "amount": -5
-          }
-        ]
+        "other_clan": {
+            "changed": -1
+        }
     },
     {
-        "event_id": "gen_misc_any_accessorykitparent1",
+        "event_id": "gen_misc_war_secret_friendship_apps_outspoken",
         "location": ["any"],
         "season": ["any"],
-        "sub_type": ["accessory"],
-        "weight": 10,
-        "event_text": "No matter how hard r_c tries, there's always something stuck in m_c's fur.",
-        "new_accessory": ["HERBS", "PETALS", "DRY HERBS", "DRY CATMINT", "DRY NETTLES", "DRY LAURELS"],
+        "sub_type": ["war"],
+        "weight": 20,
+        "event_text": "m_c is discovered gambolling along the border with an apprentice from o_c_n! Despite the reminder that the two Clans are far from allies, m_c loudly declares that {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} not giving up {PRONOUN/m_c/poss} friend over some dumb war.",
```

Comment:
a some_clan negative relationship change might be nice here, I imagine not everyone in the clan would appreciate this app's stance